README.md

```diff
@@ -51,6 +51,8 @@ import React from "react";
 import ReactDOM from "react-dom";
 import { ContextMenu, MenuItem, ContextMenuTrigger } from "react-contextmenu";
 
+import 'react-contextmenu/public/styles.5bb557.css';
```

Comment:
Probably you're right because I'm newbie on React. :)
I saw that context menu without style when I tried this component and I wanted just share my solution with you. 
I hope you can find smartest solution for that.